exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,29 @@
+function shoutyGreeting(str) {
```

Comment:
The exercise requires two functions, one that spells their names in  uppercase, and one that create a shouty greeting.
While you code works perfectly, maybe consider 
```suggestion
function shoutyGreeting(str) {  // logs the output ( shout greeting )
.....
function upperCaseName(str) { // uppercases the
```